modules/src/main/java/eu/itesla_project/modules/online/OnlineWorkflowParameters.java

```diff
@@ -114,6 +117,7 @@ public OnlineWorkflowParameters(DateTime baseCaseDate, int states, Interval hist
         this.limitReduction = limitReduction;
         this.handleViolationsInN = handleViolationsInN;
         this.constraintMargin = constraintMargin;
+        this.caseFile = caseFile;
```

Comment:
Could you try to implement an exclusive choice between the two modes: (case-date, country, case-type) ou (case-file)?